src/OpenColorIO/GpuShaderClassWrapper.cpp

```diff
@@ -5,6 +5,8 @@
 
 #include "GpuShaderClassWrapper.h"
 
+#include <cctype>
```

Comment:
It's interesting that you needed to add this.  Patrick, is it expected that GPUShaderClassWrapper.cpp is different between main and RB-2.1?

Reply:
From memory I added it in https://github.com/AcademySoftwareFoundation/OpenColorIO/pull/1515 to fix some errors in Windows latest analysis build. I think it's a good thing to have it here indeed, but also curious how you found this @fnordware, did you get some errors as well?

Reply:
Maybe best to pull the change directly from [main](https://github.com/AcademySoftwareFoundation/OpenColorIO/blob/main/src/OpenColorIO/GpuShaderClassWrapper.cpp) to avoid unneeded conflicts, the include order seems to be different with your change.

Reply:
Double-checked on macOS, this include is useless!

Reply:
Definitively needed on Windows for some MSVC versions though.

Reply:
Yeah, I got errors when building on Windows. That was RB-2.1, didn't check main as I should have. Mac OS includes a lot of headers by default, and you get a lot more if you include something like Cocoa.h. Not the first time Windows reminded me to include a header Mac didn't need.